### PR TITLE
main: update schema lease argument to duration format.

### PR DESCRIPTION
**Pull request by coocood**
Allows use to use a lease duration less than a second.

### CODE REVIEW

**Comment by coocood** [comment]
@iamxy @siddontang PTAL

**Comment by iamxy** [comment]
LGTM

File: tidb-server/main.go [hunk 1]
```diff
@@ -206,3 +203,15 @@ func prometheusPushClient(addr string, interval time.Duration) {
 		time.Sleep(interval)
 	}
 }
+
+// parseLease parses lease argument string.
+func parseLease() time.Duration {
+	dur, err := time.ParseDuration(*lease)
```
Comment (siddontang) [inline tidb-server/main.go]: how about passing 1 here?

Comment (coocood) [inline tidb-server/main.go]: then we append 's', parse '1s' will succeed.

**Comment by hanfei1991** [comment]
LGTM

**Comment by siddontang** [comment]
LGTM

**Pull request merged**